### DISCUSSION TITLE
Can't install on Windows 11 24H2 build 26.1000.1000

### DISCUSSION THREAD

**Discussion by 5T33Z0** [valinet/ExplorerPatcher#3455] [General]
I get this Error message from Prgram Compatibilty Assistant. Seems like the app is black-listed by MS:

![Screenshot 2024-07-08 204257](https://github.com/valinet/ExplorerPatcher/assets/76865553/aa9e7879-ee64-4e32-9e6f-488f425d49db)

BTW: This is a brand new install of Win11

**Comment by pyrates999** [comment]
see here: https://github.com/valinet/ExplorerPatcher/issues/3398

and duplicate of this: https://github.com/valinet/ExplorerPatcher/issues/3053
please close this.

**Comment by pyrates999** [comment]
24h2 apparently was also just released yesterday: https://en.wikipedia.org/wiki/Windows_11,_version_24H2

**Comment by 5T33Z0** [comment]
Also: the app seems to be blacklisted by MS now. It will automatically be deleted from the downloads folder if downloaded with edge followd by a notification that ep is a potential security risk.
